apps/dashboard/src/main/java/com/akto/utils/GithubSync.java

```diff
@@ -0,0 +1,68 @@
+package com.akto.utils;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.akto.log.LoggerMaker;
+import com.akto.log.LoggerMaker.LogDb;
+
+public class GithubSync {
+    private static final Logger logger = LoggerFactory.getLogger(GithubSync.class);
+    private static final LoggerMaker loggerMaker = new LoggerMaker(GithubSync.class);
+    private static final OkHttpClient client = new OkHttpClient();
+
+    public Map<String, String> syncDir(String repo, String dirPath) {
```

Comment:
please implement a syncFile function as well. Also test it out, so that it can be used without any changes for all otherfuture use cases for single file sync